src/Symfony/Bridge/Twig/TokenParser/DumpTokenParser.php

```diff
@@ -40,7 +40,7 @@ public function parse(Token $token): Node
         }
         $this->parser->getStream()->expect(Token::BLOCK_END_TYPE);
 
-        return new DumpNode($this->parser->getVarName(), $values, $token->getLine(), $this->getTag());
+        return new DumpNode($this->getVarName(), $values, $token->getLine(), $this->getTag());
```

Comment:
@fabpot Do I see it correctly that this is the only deprecation-free solution on 5.4 where we cannot change the `DumpNode` class (we could maybe make it accept a `LocalVariable` object in 7.2)?

Reply:
we can change it to accept either a string or a LocalVariable.

Reply:
I considered that too but I was wondering if doing that in a patch release is really the way to move forward.

Reply:
I did the change